make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -485,6 +484,13 @@ BUILD_LIBFONTMANAGER_FONTLIB +=  $(LIBFREETYPE_LIBS)
 
 LIBFONTMANAGER_OPTIMIZATION := HIGHEST
 
+ifneq ($(filter $(TOOLCHAIN_TYPE), gcc clang), )
```

Comment:
Can we have a note here that the de-opt is possible because these two files aren't important to OpenJDK ?
If these files were to be renamed in an updated version of harfbuzz, would there be a compilation error ?

Reply:
Comment updated.

No, these directives will be just silently ignored. So if that happens we run the risk of reintroducing the build performance regression. Not a big risk. In any case, at next harfbuzz update it can be worth re-checking if these exceptions are still needed. (If you're the one doing it and need help with that, just let me know!)

Reply:
As of today they are still there in upstream harfbuzz on github.